### ISSUE TITLE
Align comment-text correctly when image is included in comment

### ISSUE THREAD

**Issue by Junebugging** [LemmyNet/lemmy-ui#1281]
### Is your proposal related to a problem?

To demonstrate with a picture (which surprisingly is not from the marxist instance):

<img src="https://github.com/LemmyNet/lemmy-ui/assets/136490143/6539d731-0d25-4f8f-bea1-ab400d2ebc97" height="600px" />

The first image is posted with text in the comment, the last one isn't so is formatted normally
At first glance it visually looks like an abrupt jump in comment depth, as the contents of the comment appear in inconsistent places.

This seems to be a consequence of \<img> objects being placed inside \<p> objects

<img src="https://github.com/LemmyNet/lemmy-ui/assets/136490143/90793177-6964-4b9f-9f85-ceb40a125979" height="400px" />

### Describe the solution you'd like.

* Align text to the start or top somehow, so the text is placed vertically consistently
* Show text after attached images, so the image is placed horizontally consistently.

This improves the flow of reading, as you can expect the contents of the message to be placed immediately underneath the username.

Here's a quick-fix I did by making the inner-comment <p> block explicitly flex, flex-direction to column, and aligning items to start *(default is "stretch", which stretches images)*. 
![image](https://github.com/LemmyNet/lemmy-ui/assets/136490143/d6619131-6cd8-4b48-9319-33f0ff82aa12)

A rule to implement the above:
```css
.comment-node > div > div > .md-div p {
    display:          flex;
    align-items:    start;
    flex-direction: column;
}
```
Possible issues: If the comments are styled using another display-type than flex, this could mess with that (though that doesn't seem to be the case, parent objects also use flex). The CSS rule suggested above is also quite general, but it's as generic but specific as I could make it to apply to all comments.

### Describe alternatives you've considered.

Parsing the users comment input to separate \<img>'s from \<p>'s would make it easier to format them, but that requires additional parsing and additional potential surfaces for bugs.

**Comment by Junebugging** [comment]
This doesn't seem to be an issue in some cases, because the \<img> is in a separate \<p>:
![image](https://github.com/LemmyNet/lemmy-ui/assets/136490143/e62c96f7-332c-47d8-a4cc-2c163cb6a388)
I assume this is the case when entering the image on a new paragraph

**Comment by dessalines** [comment]
Correct, its dependent on image size, and the fact that you inlined it. You'd need to open an issue about this with markdown-it, our markdown parser.
